portfolio_style.css

```diff
@@ -0,0 +1,194 @@
+@import url('https://fonts.googleapis.com/css?family=Charm|Fredericka+the+Great');
+
+html {
+	font-family: "Fredericka the Great";
+	color: #5f6161;
+	font-size: 1rem;
+	
+}
+
+*, *::before, *::after {
+	margin: 0;
+	padding: 0;
+	box-sizing: border-box;
+}
+	
+body {
+	background-color: white;
+}
+
+a {
+	color: white;
+	text-align: center;
+}
+
+a:hover {
+	color: #e00dbd
+}
+
+a:focus{
+	color:#e00dbd;
+}
+
+p {
+	text-align: center;
+}
+
+h1, h2 {
+	color: #b5b0b3;
+	text-align: center;
+	font-weight: normal;
+}
+
+h1 {
+	font-size: 5rem;	
+}
+
+h3 { 
+	font-size: 2rem;
+	font-weight: normal;
+}
+
+.nav_bar {
+	display: flex;
+	flex-direction: row;
+	justify-content: space-between;
+	background-color: #e69ccb;
+	color: white;
+	height: 50px;
+	padding: 10px;
+	width: 100%;
+
+}
+
+.section {
+	display: flex;
+	flex-direction: row;
+	flex-wrap: nowrap;
+	padding-left: 50px;
+	padding-right: 50px;
+
+}
+
+.welcome {	
+	background-color: #f7daed;
+	font-size: 3rem;
+	padding-bottom: 200px;
+	height: 90%;
```

Comment:
90% refers to the height of the parent element, in this case it's the `body` tag. The `body` tag doesn't have a defined height - it is only as tall as it needs to be to fit its children, so in this case it is less than the height of the page or the window. We use % when we know that the parent has a set size. So just use some other metric for the height of this.